WTWSMS/common/scripted_triggers/00_scripted_triggers.txt

```diff
@@ -465,6 +473,9 @@ has_fertile_trait_trigger = {
 		trait = dancing_plague
 		trait = fian
 		trait = celtic_knight
+		trait = fertile
+		trait = seducer
+		trait = seductress
```

Comment:
It's simple math: Seducers are going to be "nailing everything that ain't nailed down". They will, by the sheer number of attempts they take, be exponentially more likely to conceive children.

In addition, when a fertile trigger and an infertile trigger cross, the infertile wins.